### ISSUE TITLE
Khal crashes on startup due to new priority system code

### ISSUE THREAD

**Issue by samdmarshall** [pimutils/khal#856]
Up until a few minutes ago I was using khal version `0.9.10`, and then upgraded to `0.10.0` and now it throws an exception on launch due to what appears to be the code added in #812 from the backtrace:

```
Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/bin/ikhal", line 10, in <module>
    sys.exit(main_ikhal())
  File "/home/linuxbrew/.linuxbrew/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/linuxbrew/.linuxbrew/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/linuxbrew/.linuxbrew/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/linuxbrew/.linuxbrew/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/linuxbrew/.linuxbrew/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/linuxbrew/.linuxbrew/lib/python3.7/site-packages/khal/cli.py", line 481, in interactive_cli
    multi_calendar_select(ctx, include_calendar, exclude_calendar)
  File "/home/linuxbrew/.linuxbrew/lib/python3.7/site-packages/khal/cli.py", line 165, in build_collection
    'priority': cal['priority'],
  File "/home/linuxbrew/.linuxbrew/lib/python3.7/site-packages/configobj-5.0.6-py3.7.egg/configobj.py", line 554, in __getitem__
KeyError: 'priority'
```
my khal config has a bunch of calendar entries, none of which contain a `priority` property/key. I did test out adding the key with the default setting to all of my calendars just to see if that fixed it and it continues to get hung-up on this error. My config is back to not including the new `priority` keys and for now gonna just roll back to 0.9.10 via pip as that still seems to work okay.

**Comment by geier** [comment]
Can you share the structure of your config file? (Feel free to anonymise the actual paths, etc.) For the moment, I can't reproduce this on my end.

**Comment by samdmarshall** [comment]
https://github.com/samdmarshall/dotfiles/blob/linux/khal/config

**Comment by geier** [comment]
@samdmarshall thanks! can confirm, will fix and then do a new release.

**Comment by geier** [comment]
fixed and released v0.10.1

**Comment by samdmarshall** [comment]
Awesome! thanks so much.
